### PR TITLE
Update documentation to be milliseconds

### PR DESCRIPTION
**Pull request by kmadsen**
## Description

Update documentation to represent the value. Confusion for our timestamps comes from the different types of representation we have.

Directions gives time in the `second Double form`. e.g., 61.245 is a little over a minute.
Route progress and navigation native is `millisecond Long form`. e.g., 61245L is a little over a minute

Note that the old [RouteProgress is in second Double form](https://github.com/mapbox/mapbox-navigation-android/blob/1065241164ffb936ab59b301ffbfa8a1d594c423/libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/routeprogress/RouteProgress.kt#L29)

- [x] I have added any issue links
- [ ] I have added all related labels (`bug`, `feature`, `new API(s)`, `SEMVER`, etc.)
- [ ] I have added the appropriate milestone and project boards

## Testing

Please describe the manual tests that you ran to verify your changes

- [ ] I have tested locally (including `SNAPSHOT` upstream dependencies if needed) through testapp/demo app and run all activities to avoid regressions
- [ ] I have tested via a test drive, or a simulation/mock location app
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have updated the `CHANGELOG` including this PR

### CODE REVIEW

File: libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/RouteProgress.kt [hunk 1]
```diff
@@ -41,10 +41,9 @@ class RouteProgress private constructor(
     fun distanceTraveled(): Float = distanceTraveled
 
     /**
-     * Provides the duration remaining in seconds till the user reaches the end of the route.
+     * Provides the duration remaining in milliseconds till the user reaches the end of the route.
```
Comment (kmadsen) [inline libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/RouteProgress.kt]: actually didn't realize that it was the documentation that threw me off too
https://github.com/mapbox/mapbox-navigation-android/pull/2569/files#diff-d63e04e6e9257f3b5b59fcaef29b39b6R49

**Comment by kmadsen** [comment]
There may be an inconsistency between Nav Native and Directions. I'm looking for it now

**Comment by Guardiola31337** [comment]
This is a regression from https://github.com/mapbox/mapbox-navigation-android/pull/2365 and https://github.com/mapbox/mapbox-navigation-android/pull/2378

Legacy handled this differently 👀 https://github.com/mapbox/mapbox-navigation-android/blob/1065241164ffb936ab59b301ffbfa8a1d594c423/libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/routeprogress/RouteProgress.kt#L96-L106 and now we're including the `durationRemaining` in the `RouteProgress` https://github.com/mapbox/mapbox-navigation-android/blob/1065241164ffb936ab59b301ffbfa8a1d594c423/libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/RouteProgress.kt#L20 and getting the value directly from NN (`NavigationStatus`) 👀 https://github.com/mapbox/mapbox-navigation-android/blob/1065241164ffb936ab59b301ffbfa8a1d594c423/libnavigator/src/main/java/com/mapbox/navigation/navigator/MapboxNativeNavigatorImpl.kt#L181-L187 https://github.com/mapbox/mapbox-navigation-android/blob/1065241164ffb936ab59b301ffbfa8a1d594c423/libnavigator/src/main/java/com/mapbox/navigation/navigator/MapboxNativeNavigatorImpl.kt#L189

We're doing the conversion to seconds for `RouteLegProgress` https://github.com/mapbox/mapbox-navigation-android/blob/1065241164ffb936ab59b301ffbfa8a1d594c423/libnavigator/src/main/java/com/mapbox/navigation/navigator/MapboxNativeNavigatorImpl.kt#L264 and `RouteStepProgress` https://github.com/mapbox/mapbox-navigation-android/blob/1065241164ffb936ab59b301ffbfa8a1d594c423/libnavigator/src/main/java/com/mapbox/navigation/navigator/MapboxNativeNavigatorImpl.kt#L260 For consistency, should we do the same for `RouteProgress` @kmadsen? Should we remove these conversions? One or the other 😉 What do you think @kmadsen?

**Comment by kmadsen** [comment]
Looks like there is a separate bug in `MapboxNativeNavigatorImpl`. 

Whenever there are 2 or more legs, the duration will basically not change. (adding 60 seconds to 60,000 milliseconds will only be 60,060 milliseconds 😆 )

Considering that we don't have many tests set up for 2 or more legs (also why we haven't noticed this), and the `MapboxNativeNavigatorImpl` has no tests - I prefer to handle that issue separately.

**Comment by Guardiola31337** [comment]
In any case, we're doing the conversion (to seconds) for `RouteLegProgress` and `RouteStepProgress` but not for `RouteProgress`. I believe we should do one or the other, independent if there's a bug in the multi-legs scenario.

**Comment by kmadsen** [comment]
> In any case, we're doing the conversion (to seconds) for RouteLegProgress and RouteStepProgress but not for RouteProgress. I believe we should do one or the other, independent if there's a bug in the multi-legs scenario.

Yeah I agree and it's one or the other. My proposal is to fix documentation because it is the current issue. We will introduce new issues with new changes.

Current status: Documentation says seconds when it is milliseconds
1 known issue: If route has 2 or more legs, the ETA will only represent the current leg.

Possible change 1: Update documentation to say milliseconds instead of seconds
 -- Need to fix routes with 2 or more legs

Possible change 2: Update durationRemaining to be in Double second form
 -- Need to fix FasterRoute
 -- Need to update MapboxNavigationTelemetry

**Comment by kmadsen** [comment]
Follow up for creating routes with multiple legs https://github.com/mapbox/mapbox-navigation-android/issues/2665

**Pull request merged**